config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -225,7 +225,7 @@ presubmits:
             # Panic if anything mutates a shared informer cache
             - --env=ENABLE_CACHE_MUTATION_DETECTOR=true
             - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce
-            - --test_args=--ginkgo.skip=\[Driver:.gcepd\]|\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
+            - --test_args=--ginkgo.skip=\[Driver:.gcepd\]|\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[Beta\]|\[Alpha\] --minStartupPods=8
```

Comment:
just do `|\[FeatureGate:.+\]` that was the spirit

Reply:
The following words are pasted from slack (@pohly said):

Regarding skipping on FeatureGate: I think the intent will be clearer when skipping on [Beta] and [Alpha] - the effect is the same, it's just clearer what the connection is to jobs which then focus on those tags.

Reply:
Please see https://github.com/kubernetes/test-infra/issues/31666 for detail context